index.html

```diff
@@ -112,6 +112,24 @@ <h3 class="works-title yelow abt-me">About me</h3>
   <li>Terminal</li>
   </ul>
   </ul>
-  </body>
+  </section>
 
+  </body>
+  <footer class="form-section">
+    <h2 class="contact-title font-montesori">I'm always <br>interested in hearing <br>about new projects, <br>so if you'd like to <br>chat please get in touch.</h2>
+    <form action="https://formspree.io/f/mwkzgoyj" method="post" class="form-container">
+    <input type="text" name="Name" id="name" placeholder="Full name"  maxlength="30" class="input-form" required>
+    <input type="email" name="Email" id="email" placeholder="Email address" class="input-form" required>
+    <textarea  name="Message" id="message" cols="30" rows="6" placeholder="Enter text here" maxlength="500" class="form-text-area" required></textarea>
+    <button type="submit" class="form-button">Get in touch</button>
+    </form>
+    <ul class="form-section-media-icons">
+    <li><a href="https://github.com/aubinleyoung"><img src="/assets/icons/Vector2.svg" alt=""></a></li>
+    <li><a href="https://www.linkedin.com/in/aubin-simpeze-7a5b7a220/"><img src="/assets/icons/Linkedin icon.svg" alt=""></a></li>
+    <li><a href=""><img src="/assets/icons/Vector3.svg" alt=""></a></li>
+    <li><a href="https://twitter.com/SimpezeAubin"><img src="/assets/icons/Vector.svg" alt=""></a></li>
+    <li><a href=""><img src="/assets/icons/Vector1.svg" alt=""></a></li>
+    </ul>
```

Comment:
- [ ] You did a great job working on this milestone! 🚀 However, please consider fixing the paths to your icons, at the moment they're not displayed properly:

```suggestion
    <ul class="form-section-media-icons">
    <li><a href="https://github.com/aubinleyoung"><img src="./assets/icons/Vector2.svg" alt=""></a></li>
    <li><a href="https://www.linkedin.com/in/aubin-simpeze-7a5b7a220/"><img src="./assets/icons/Linkedin icon.svg" alt=""></a></li>
    <li><a href=""><img src="./assets/icons/Vector3.svg" alt=""></a></li>
    <li><a href="https://twitter.com/SimpezeAubin"><img src="./assets/icons/Vector.svg" alt=""></a></li>
    <li><a href=""><img src="./assets/icons/Vector1.svg" alt=""></a></li>
    </ul>
```


Please consider adding the dot at the beginning to make it a relative path! 🤓 🛠️ 

- [ ] **[OPTIONAL]:** Please consider fixing indentation as this will keep your code clean and improve its readability! I strongly suggest you apply this change! 🤓